perf/README.md

```diff
@@ -15,24 +15,49 @@ Benchmark results can be visualized with https://observablehq.com/@libp2p-worksp
 3. Wait for action run to finish and to push a commit to your branch.
 4. Visualize results on https://observablehq.com/@libp2p-workspace/performance-dashboard.
 
-## Running via local machine
+## Running with Terraform on AWS manually
 
+### Pre-requisites
+
+- Terraform 1.5.5 or later
+- Node.js 18 or later
+- [an AWS IAM user](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users.html)
+##
 ### Provision infrastructure
 
-1. Save your public SSH key as the file `./terraform/modules/short_lived/files/perf.pub`; or generate a new key pair with `make ssh-keygen` and add it to your SSH agent with `make ssh-add`.
-2. `cd terraform/configs/local`
-3. `terraform init`
-4. `terraform apply`
-5. `CLIENT_IP=$(terraform output -raw client_ip)`
-6. `SERVER_IP=$(terraform output -raw server_ip)`
+1. Ensure you ssh agent is running, you can check that via
+```bash
+   eval "$(ssh-agent -s)"
+```
```

Comment:
I don't think SSH requires any background tasks.

Reply:
If I try to run [make ssh-add](https://github.com/libp2p/test-plans/blob/master/perf/Makefile#L4) without running an ssh-agent I will get the error

```
Could not add identity "./terraform/modules/short_lived/files/perf": agent refused operation
make: *** [ssh-add] Error 1
```

Reply:
It’s not a requirement for using SSH.
I don’t think we need to point out utterly trivial things.